contracts/cw20-atomic-swap/src/state.rs

```diff
@@ -23,12 +23,12 @@ impl AtomicSwap {
     }
 }
 
-pub const SWAPS: Map<&str, AtomicSwap> = Map::new("atomic_swap");
+pub const SWAPS: Map<String, AtomicSwap> = Map::new("atomic_swap");
```

Comment:
There's no need to change this. Borrowed keys are supported just fine. And in fact, it's recommended to use it, o avoid extra clones.